integrated_channels/degreed/models.py

```diff
@@ -132,6 +99,40 @@ class DegreedEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfigurati
         help_text="The organization code provided to the enterprise customer by Degreed."
     )
 
+    degreed_base_url = models.CharField(
+        max_length=255,
+        blank=True,
+        verbose_name="Degreed Base URL",
+        help_text="The base URL used for API requests to Degreed, i.e. https://degreed.com."
+    )
+
+    degreed_user_id = models.CharField(
+        max_length=255,
+        blank=True,
+        verbose_name="Degreed User ID",
+        help_text=(
+            "The Degreed User ID provided to the content provider by Degreed. "
+            "It is required for getting the OAuth access token."
+        )
+    )
+
+    degreed_user_password = models.CharField(
+        max_length=255,
+        blank=True,
+        verbose_name="Degreed User Password",
+        help_text=(
+            "The Degreed User Password provided to the content provider by Degreed. "
+            "It is required for getting the OAuth access token."
+        )
+    )
+
+    provider_id = models.CharField(
+        max_length=100,
+        default='EDX',
```

Comment:
Not sure we should have a default here, from an Open edX perspective -- it probably should just be a field that accepts null values.